db/Dockerfile

```diff
@@ -1,7 +1,18 @@
+# Debian variant
 FROM postgres
```

Comment:
we should probably make this a specific version since we're messing with internal stuff

Reply:
ok